src/components/Contacts/ContactsTable/ContactsTable.tsx

```diff
@@ -14,11 +14,17 @@ import { useContactsQuery } from '../../../../pages/accountLists/[accountListId]
 interface Props {
   accountListId: string;
   onContactSelected: (contactId: string) => void;
+  activeFilters: boolean;
+  filterPanelOpen: boolean;
+  toggleFilterPanel: () => void;
 }
 
 export const ContactsTable: React.FC<Props> = ({
   accountListId,
   onContactSelected,
+  activeFilters,
+  filterPanelOpen,
+  toggleFilterPanel,
```

Comment:
Is this the standard you guys have for deconstructing props in the project?

Reply:
Ya I think we usually try and use destructuring vs having `props.` everywhere.